flink-runtime/src/main/java/org/apache/flink/runtime/security/token/DefaultDelegationTokenManager.java

```diff
@@ -120,8 +135,8 @@ private Map<String, DelegationTokenProvider> loadProviders() {
             } catch (Exception | NoClassDefFoundError e) {
                 LOG.warn(
                         "Failed to initialize delegation token provider {}",
-                        provider.serviceName(),
-                        e);
+                        provider.serviceName());
+                LOG.debug(e.toString());
```

Comment:
Add some explanation to this log message since given the multi-threaded nature of Flink it might not directly follow the previous line.

Reply:
This part has beeen changed significantly so I think worth a second look.